app/src/main/java/org/simple/clinic/drugs/search/DrugSearchModel.kt

```diff
@@ -1,9 +1,17 @@
 package org.simple.clinic.drugs.search
 
-class DrugSearchModel {
+data class DrugSearchModel(
+    val searchQuery: String?
```

Comment:
Instead of a nullable string, should it be non-nullable and the default value be a blank string instead?

Reply:
Yeah, good point. We can do the `orEmpty` extension at the UI level rather than at `Update`. Currently, I am doing that in `Update` function to decide search or do nothing.